analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantDeclaration.ArraySize.Fixed.cs

```diff
@@ -7,16 +7,16 @@ abstract class RedundantDeclaration
     {
         public RedundantDeclaration()
         {
-            MyEvent += new EventHandler((a, b) => { }); // Fixed
-            MyEvent += (a, b) => { };
+            MyEvent += new EventHandler(Handle); // Fixed
+            MyEvent += (a, b) => { Handle(a, b); };
```

Comment:
I've changed the test cases to use the parameters from declarations to avoid different results when running for C# 9 language vers.

Reply:
do we have the same test case (unused lambda params) to run for both C# 8 and C# 9? 

It feels it's valuable to have the same test case running on two different language versions, to verify for regressions and make the behavior explicit from the tests.

At least a small, minimal example which gets analyzed and fixed differently with C# 8 / C# 9.